### ISSUE TITLE
Documentation for "New" stuff

### ISSUE THREAD

**Issue by ahlstromcj** [jackaudio/new-session-manager#98]
I've been testing a Seq66 NSM-related fix on an Ubuntu system, a bit behind the times.  On that system, there are two things to consider:  (1) the legacy "NSM sessions" directory is still in use; (2) the "Resume" message doesn't seem to restart the qseq66 application.  This should be documented as a trouble-shooting procedure, I think.

I am now running another laptop using Arch Linux, and here is how it acts vice the two points noted above: (1) The sessions directory has been moved to ".local/share/nsm" (would .config/nsm be a good alternative?); (2) The "Stop" and "Resume" options work as I would expect on my Arch Linux laptop,

So, minor documentation issues, and congratulations on bringing NSM up-to-date.

Side note: I'm adding the directory change to the Seq66 documentation.

Huzzah!

**Comment by mxmilkiib** [comment]
Beyond https://new-session-manager.jackaudio.org/api/index.html#_api_versions_and_behaviour_changes? I guess it only mentions the path has changed there, but the actual new path is only mentioned above in the main API section.

**Comment by ahlstromcj** [comment]
So a couple things:  (1) man pages! Whoodathunkit!!? (2) /usr/share/doc/new-session-manager! Same thing!

It totally did not occur to that the jackaudio guy(s) would make a standard UNIX install out of NSM. 

So, while blushing, I will close this premature issue report. I really appreciate all the work that went into this fork. Glad I wiped Ubuntu from my little laptop and added Arch, to get more up-to-date stuff.

As an aside, I added a function to the Seq66 daemonize module to determine the parent of qseq66/seq66cli before trying to read the stock configuration files in ~/.config/seq66. Thus a new nsm session will start clean.  Still working through configuration import....
